packages/valory/skills/termination_abci/rounds.py

```diff
@@ -65,19 +66,12 @@ def termination_majority_reached(self) -> bool:
         """Get termination_majority_reached."""
         return cast(bool, self.db.get("termination_majority_reached", False))
 
-    @property
```

Comment:
Already exists at the base

Reply:
I don't think this removal is right. The base one uses `self.participants` which uses `get_strict` instead of `get`. So this method would raise a `ValueError` when the list of participants is not set. Which can happen since the BackgroundRound doesn't depend on any of the other rounds finishing.

Reply:
This has been changed on the base now. You are right but the base implementation was not ideal. Your override is the more appropriate solution as we should also be able to use the property with 0 participants.